tests/integration_tests/performance/test_process_startup_time.py

```diff
@@ -10,7 +10,7 @@
 import host_tools.logging as log_tools
 from host_tools.cargo_build import run_seccompiler_bin
 
-MAX_STARTUP_TIME_CPU_US = {'x86_64': 5500, 'aarch64': 3100}
```

Comment:
So far I've never seen the tests failing with a value grater than `3170`. Based of how many test iterations was this limit computed?

Reply:
I did a lot of iterations while trying to find the root cause of the issue and I also saw values a little over 3400 (but that was very scarce). Also I do not think it is sustainable to increase it just by a little bit every time. One other observation was that on 4.14  x86, the value is way below 5500.

```
================================================================================ 7 passed in 7.85s =================================================================================
[ec2-user@ip-172-31-21-17 firecracker]$ ./tools/devtool test -- -s integration_tests/performance/test_process_startup_time.py
[Firecracker devtool] 2022-01-12 11:23:39 UTC
[Firecracker devtool] Starting test run ...
=============================================================================== test session starts ================================================================================
platform linux -- Python 3.6.9, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
cachedir: ../build/pytest_cache
rootdir: /firecracker/tests, configfile: pytest.ini
plugins: timeout-1.4.2
timeout: 300.0s
timeout method: signal
timeout func_only: False
collected 3 items                                                                                                                                                                  

[ performance | 3 tests | 1 worker(s) ]
Process startup time is: 26514 us (2863 CPU us)
  PASSED    integration_tests/performance/test_process_startup_time.py::test_startup_time_new_pid_ns[ubuntu]
Process startup time is: 22194 us (2824 CPU us)
  PASSED    integration_tests/performance/test_process_startup_time.py::test_startup_time_daemonize[ubuntu]
Process startup time is: 21237 us (3052 CPU us)
  PASSED    integration_tests/performance/test_process_startup_time.py::test_startup_time_custom_seccomp[ubuntu]

================================================================================ 3 passed in 6.31s =================================================================================
[ec2-user@ip-172-31-21-17 firecracker]$ uname -a
Linux ip-172-31-21-17.us-east-2.compute.internal 4.14.252-195.483.amzn2.x86_64 #1 SMP Mon Nov 1 20:58:46 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```